src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/ColorPickerTests/WinUIColorPickerPage.xaml.cs

```diff
@@ -389,8 +389,13 @@ private void RTLCheckBox_Unchecked(object sender, RoutedEventArgs e)
             this.ColorPicker.FlowDirection = FlowDirection.LeftToRight;
         }
 
+		private void OrientationComboBox_SelectionChanged(object sender, SelectionChangedEventArgs e)
+		{
+			this.ColorPicker.Orientation = ((e.AddedItems[0] as ComboBoxItem).Content as string) == "Horizontal" ? Orientation.Horizontal : Orientation.Vertical;
+		}
+
 		// Uno TODO: Move this out into a helper class
-        public static DependencyObject FindVisualChildByName(FrameworkElement parent, string name)
+		public static DependencyObject FindVisualChildByName(FrameworkElement parent, string name)
         {
```

Comment:
Somehow indentation is all broken up again.

Reply:
This happens because the code is copy/pasted from upstream including the spaces for indentation. As you are adding the code by hand it's simply appying tabs from the editorconfig. My recomendation: just copy/paste the entire file. Don't change anything and don't edit by hand.